### ISSUE TITLE
admins with @css/root don't work, but @css/generic does

### ISSUE THREAD

**Issue by ar1a** [shobhit-pathak/MatchZy#37]
with my admin file setup like so

```json
{
  "ar1a": {
    "identity": "76561198199062669",
    "flags": ["@css/root"]
  }
}
```

matchzy doesn't recognize me as an admin, if i change it to `@css/generic`, it does

**Comment by shobhit-pathak** [comment]
Hey!

This is because `@css/generic` flag is required for MatchZy admin. All the admin commands currently fall under the `@css/generic` flag only.

Discussion for adding granular support is going on here: https://github.com/shobhit-pathak/MatchZy/issues/33

**Comment by ar1a** [comment]
From my understanding of the docs, @css/root should grant you all flags, including @css/generic

**Comment by shobhit-pathak** [comment]
Thanks for pointing it out, I might have missed it!

I'm working on improving the admin system, currently I had used a workaround to detect an admin. But now I'll use the proper way to define the admin commands :D

**Comment by crashzk** [comment]
There is also a discussion about the `@css/root` flag here: https://github.com/roflmuffin/CounterStrikeSharp/issues/125

If it is implemented, I believe it will solve this problem,

**Comment by ar1a** [comment]
that explains all the confusion!

**Comment by shobhit-pathak** [comment]
Hey!

This has been fixed in the 0.4.3-alpha release, thanks! :D
